core/config/locales/en.yml

```diff
@@ -1282,7 +1282,7 @@ en:
     inventory: Inventory
     inventory_adjustment: Inventory Adjustment
     inventory_canceled: Inventory canceled
-    inventory_error_flash_for_insufficient_quantity: An item in your cart has become unavailable.
+    inventory_error_flash_for_insufficient_quantity: "'#{names}' has become unavailable."
```

Comment:
Seems like quoting the outside of the sentence will read quite odd if multiple items are out of stock.

> 'blue shirt and red shirt' has become unavailable.